sdk/core/azure-core/src/main/java/com/azure/core/util/polling/PollerFlux.java

```diff
@@ -38,68 +38,143 @@
-     *                             operation will be called current {@link PollingContext}, if service does not have an
-     *                             api to fetch final result and if final result is same as final poll response value
-     *                             then implementer can choose to simply return value from provided final poll response.
+     *     the long running operation if service support it. This parameter is required and operation will be called
+     *     current {@link PollingContext}. If service does not have an api to fetch final result and if final result
+     *     is same as final poll response value then implementer can choose to simply return value from provided
+     *     final poll response.
      */
-    public PollerFlux(Duration defaultPollInterval,
+    public PollerFlux(Duration pollInterval,
                       Function<PollingContext<T>, Mono<T>> activationOperation,
                       Function<PollingContext<T>, Mono<PollResponse<T>>> pollOperation,
                       BiFunction<PollingContext<T>, PollResponse<T>, Mono<T>> cancelOperation,
                       Function<PollingContext<T>, Mono<U>> fetchResultOperation) {
-        Objects.requireNonNull(defaultPollInterval, "'defaultPollInterval' cannot be null.");
-        if (defaultPollInterval.compareTo(Duration.ZERO) <= 0) {
+        Objects.requireNonNull(pollInterval, "'pollInterval' cannot be null.");
+        if (pollInterval.compareTo(Duration.ZERO) <= 0) {
             throw logger.logExceptionAsWarning(new IllegalArgumentException(
                 "Negative or zero value for 'defaultPollInterval' is not allowed."));
         }
-        this.defaultPollInterval = defaultPollInterval;
-        this.activationOperation = Objects.requireNonNull(activationOperation,
-            "'activationOperation' cannot be null.");
-        this.oneTimeActivationMono = oneTimeActivationMono(activationOperation);
+        this.defaultPollInterval = pollInterval;
+        Objects.requireNonNull(activationOperation, "'activationOperation' cannot be null.");
         this.pollOperation = Objects.requireNonNull(pollOperation, "'pollOperation' cannot be null.");
         this.cancelOperation = Objects.requireNonNull(cancelOperation, "'cancelOperation' cannot be null.");
         this.fetchResultOperation = Objects.requireNonNull(fetchResultOperation,
             "'fetchResultOperation' cannot be null.");
+        this.oneTimeActivationMono = new OneTimeActivation<>(this.rootContext,
+            activationOperation,
+            // mapper
+            activationResult -> new PollResponse<>(LongRunningOperationStatus.NOT_STARTED, activationResult)).getMono();
+        this.syncActivationOperation =
+            cxt -> new PollResponse<>(LongRunningOperationStatus.NOT_STARTED, activationOperation.apply(cxt).block());
+    }
+
+    /**
+     * Creates PollerFlux.
```

Comment:
Provide a better first sentence, as this is the most visible text in the JavaDoc.